students/jeff_shabani/session04/trigrams.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import random
+import re
+
+word_reg = re.compile(r'\W+')  # regex to split text by white space
+
+
+# read in the text file and parse the words
+def trigram(file):
+    pair_dict = dict()
+    word_list = []
+
+    with open(file, 'rt') as infile:
```

Comment:
`file` is a python keyword, need to use something else

Reply:
Fixed